_data/fellow_projects/pymela.yml

```diff
@@ -0,0 +1,11 @@
+open: yes
+postdate: 2020-10-23
+title: "**pyMELA: Approximating the matrix element method integral with deep learning and automatic differentiation**"
+description: "The Matrix Element Method (MEM) is a powerful technique for high energy physics, but is highly computationally expensive.
+Recent advancements in applying deep learning to approximate the MEM integral through regression have shown dramatic computational time reduction.
+The goal of this fellowship project would be to explore ways to improve upon the established deep learning techniques through contributions to the JAX based pyMELA framework.
+If time allowed, the project could also incorporate the study of the effects of using automatic differentiation to improve the interpolation of grid points."
+
+contacts:
+ - msneubauer
+ - matthewfeickert
```

Comment:
@davidlange6 @henryiii Any thoughts on why when I build this locally it only shows Mark's name and not mine, but in PR #477 @davidlange6 was able to get two names to show up?

Reply:
Could you use normal `|` syntax here instead of long quoted strings? Also, why isn't the "bold" part of the title, instead of being placed in each title manually?

Anyway, the issue is contacts need to have emails defined in their profiles, otherwise they don't show up.

Reply:
> Anyway, the issue is contacts need to have emails defined in their profiles, otherwise they don't show up

Thanks, that should be resolved now. 

> Could you use normal `|` syntax here instead of long quoted strings? Also, why isn't the "bold" part of the title, instead of being placed in each title manually?

I'm not sure what you mean by either of these things, but I'll look into them and address try to address them both and then ask for your review.

Reply:
Hm. When I build locally after adding the emails to our profiles it still fails to add them

![contacts_empty](https://user-images.githubusercontent.com/5142394/97459946-4ac93600-190a-11eb-958d-39c631cade80.png)

Reply:
Ah okay, I now understand the `|` syntax after looking at my CI workflows. Sorry. I'm still not sure what you mean @henryiii by

>  Also, why isn't the "bold" part of the title, instead of being placed in each title manually?